desk/app/chat.hoon

```diff
@@ -671,14 +763,14 @@
       di-abet:(di-ingest-diff:(di-abed-soft:di-core p.action) q.action)
     di-abet:(di-proxy:(di-abed-soft:di-core p.action) q.action)
   ::
-      %chat-dm-diff-1
+      ?(%chat-dm-diff-1 %chat-dm-diff-2)
     =+  !<(=diff:dm:c vase)
     di-abet:(di-take-counter:(di-abed-soft:di-core src.bowl) diff)
   ::
       %chat-club-create
     cu-abet:(cu-create:cu-core !<(=create:club:c vase))
   ::
-      %chat-club-action-1
+      ?(%chat-club-action-1 %chat-club-action-2)
     =+  !<(=action:club:c vase)
```

Comment:
Are you sure these are correct? Do these marks actually have the same type? Why?